webpack.config.babel.js

```diff
@@ -16,18 +16,9 @@ export default () => (
             }
         },
         module: {
-            rules: [{
-                test: /.js$/,
-                exclude: /node_modules/,
-                use: 'babel-loader',
-                options: {
-                    presets: ['es2015-webpack']
-                }
-            }, {
-                test: /.json$/,
-                exclude: /node_modules/,
-                use: 'json-loader'
-            }]
-        }
+            rules: [
+                {test: /\.js$/, exclude: /node_modules/, loader: "babel-loader"}
```

Comment:
Oh! Thanks for teaching me that :) 
I only referred to the [schema](https://github.com/webpack/webpack/blob/master/schemas/webpackOptionsSchema.json) where I think it means both `loader` and `use` can be used interchangeably. Anyway, will try to find out the reasoning. Meanwhile, merging this now. 
Thanks!